Update debugbar dependency

rvm-peercode opened the pull request:
Closes: #16

rvm-peercode committed:
composer: Depend on php-debugbar/php-debugbar instead of maximebf/debugbar

The package maximebf/debugbar is abandoned and no longer maintained. The
author suggests using the php-debugbar/php-debugbar package instead.
Keep the version spec as it was.

rvm-peercode committed:
Replace references to maximebf/debugbar in readme

oscarotero commented:
Thank you!

The workflow file is outdated and it's failing. Could you update it with the same content of this? https://github.com/middlewares/aura-router/blob/master/.github/workflows/main.yaml

The new version not only has the latest versions of the used actions, but also tests for php 8.3 and 8.4.

rvm-peercode committed:
github: Update workflow

Taken from:
  https://raw.githubusercontent.com/middlewares/aura-router/refs/heads/master/.github/workflows/main.yaml

rvm-peercode committed:
github: Reinstate coverage

rvm-peercode committed:
github: Don't use composer.lock in cache key

We don't have that file here.

oscarotero commented:
Sorry again, the tests are still failing due old dependencies in the composer.json files. Especially diactoros. Can you change the dependencies to allow the latest versions? You can use this as reference: https://github.com/middlewares/aura-router/blob/master/composer.json#L25

rvm-peercode committed:
composer: Update dev dependencies

As per Óscar's request.

Based on:
  https://github.com/middlewares/aura-router/blob/master/composer.json

oscarotero commented:
Thank you!
